test/common/http/header_map_impl_test.cc

```diff
@@ -424,9 +424,8 @@ TEST_P(HeaderMapImplTest, AllInlineHeaders) {
     INLINE_REQ_RESP_STRING_HEADERS(TEST_INLINE_STRING_HEADER_FUNCS)
   }
   {
-    // No request trailer O(1) headers.
-  }
-  {
+      // No request trailer O(1) headers.
+  } {
```

Comment:
Please fix format here. Envoy provides some tool scripts and you may run `./ci/run_envoy_docker.sh './ci/do_ci.sh fix_format'` to fix it.